src/core/Footer/Footer.tsx

```diff
@@ -19,15 +20,13 @@ export type FooterProps = {
   /**
    * Customize footer elements.
```

Comment:
Some additional description would be useful. Can have even example in here (or in `Footer` description below

Reply:
Fixed

Reply:
Hover over the prop to see, how it is displayed now:
![image](https://user-images.githubusercontent.com/81580355/155474952-d943e4b6-06f9-4d24-b8ea-0499038b4d9d.png)
Examples are not visible.

I suggest in description wrapping `FooterElements` with char ` for better readability that it is type.

Also, examples do not require anything additional, just:

```
* @example Returning only custom elements
   * <Footer customElements={() => newFooterElements)} />
   * @example Filtering out a specific element
   * <Footer customElements={(defaultElements: FooterElement[]) => defaultElements.filter(({ key }) => key !== 'privacy' )} />
   * @example Changing a url
   * <Footer customElements={(defaultElements: FooterElement[]) => defaultElements.map(element => ({ ...element, url: element.key === 'privacy' ? customPrivacyUrl : element.url }))} />
```

And I suggest move examples to whole component description, to have a list of those there.

Reply:
Huh, the examples show up for me. You're looking at it in VS Code?

I wrapped `FooterElement[]` and moved examples to component description.

Wrapping the examples with ``` does makes a difference, though. Here the first example has the description included as part of the code, while the second example does not. The word `default` is highlighted because TSDoc is treating that line as code.
![image](https://user-images.githubusercontent.com/37936169/155615422-212ac298-9ec2-4a5c-a71d-99574f41f25e.png)

Reply:
Yes, vs code

Reply:
You current syntax is just wrong.
https://jsdoc.app/tags-example.html

This is official docs. For description you can use `<caption>`, so please change it, because current one shows nothing. Maybe you have some addition tools installed in vscode, so its shown like that.

Reply:
Oh, I thought we used TS Doc.

Reply:
TSDoc is a library that needs to be manually used. JSDoc is built into the language and IDEs.

Reply:
Never thought about tsdoc (and I have never heard of it, to be honest :D), but I dont think it would work for us, cause we deliver js code, so component should be described in jsdoc (thinking logically)

Reply:
@elephantcatdog as this was confusion, which we didnt think might happen, could you create a PR for updating our docs, that we use jsdoc? Maybe could add some more info, that every prop should have description and component should have examples...

Reply:
As far as I can tell, I don't seem to have TSDoc manually added, though it might be a part of an extension I have. But it still works for me. But yes, I will add that to the documentation.

Reply:
Now perfect.